script/release/ci-release-build.ts

```diff
@@ -189,15 +206,15 @@ async function callAppVeyor (targetBranch, job, options) {
       commitId: options.commit || undefined,
       environmentVariables
     }),
-    method: 'POST'
+    method: <const>'POST'
```

Comment:
IMO the `<const>` syntax is a bit awkward and doesn't align with the rest of TS. I think this would be more readable as `'POST' as const`, which aligns better with type assertions.

Reply:
I did some research on this because I've always just chosen a random one.

* `<const>` is a type declaration, basically a hack to specify `readonly` in an implicit type
* `as const` declares a const literal, you should prefer this one